src/components/AutoCompleteSuggestions/BaseAutoCompleteSuggestions.js

```diff
@@ -43,13 +44,25 @@ const BaseAutoCompleteSuggestions = (props) => {
         </Pressable>
     );
 
-    const rowHeight = measureHeightOfSuggestionRows(props.suggestions.length, props.isSuggestionPickerLarge);
     const innerHeight = CONST.AUTO_COMPLETE_SUGGESTER.ITEM_HEIGHT * props.suggestions.length;
 
+    const animatedStyles = useAnimatedStyle(() => ({
+        ...styles.autoCompleteSuggestionsContainer,
+        ...StyleUtils.getAutoCompleteSuggestionContainerStyle(rowHeight.value, props.shouldIncludeReportRecipientLocalTimeHeight),
+    }));
+
+    useEffect(() => {
+        rowHeight.value = withTiming(measureHeightOfSuggestionRows(props.suggestions.length, props.isSuggestionPickerLarge), {
+            duration: 100,
+            easing: Easing.inOut(Easing.ease),
+        });
+    }, [props.suggestions.length, props.isSuggestionPickerLarge, rowHeight]);
+
     return (
-        <View
+        <Animated.View
             ref={props.forwardedRef}
-            style={[styles.autoCompleteSuggestionsContainer, StyleUtils.getAutoCompleteSuggestionContainerStyle(rowHeight, props.shouldIncludeReportRecipientLocalTimeHeight)]}
+            style={animatedStyles}
+            exiting={FadeOutDown.duration(100)}
```

Comment:
Why we are adding an existing animation here? Can you upload two videos with/without the existing animation?

Reply:
Without exit animation the popup gets hidden abruptly if we clear the input.  Fadeout animation smooths it out a bit.


**without FadeOut**

https://github.com/Expensify/App/assets/6880914/7bd908e5-0c72-4cbc-ba7b-ab8ce241af39

**with FadeOut**

https://github.com/Expensify/App/assets/6880914/87f43837-c7ba-47d4-a02c-8eb1994907fe

Reply:
Let's ~annoy~ ask @shawnborton :grin:  Thoughts here?

Reply:
I don't mind the fade, but why can't we exit it the same way it enters? That feels more consistent to me.

Reply:
When we clear the input the suggestions popup gets removed from layout at `ReportActionCompose` level. For entry and further change in height of popup we are animating the container height variable. But for exit we wouldn't get an opportunity to animate the container height further, as the component gets removed from layout hierarchy immediately. That's why I'm using the `exiting` prop with `FadeOutDown` animation which feels closer to the entry animation. Other [exit animation options](https://docs.swmansion.com/react-native-reanimated/docs/api/LayoutAnimations/exitAnimations/#slide) are Bounce, Flip, Stretch, Zoom, Slide, LightSpeed, Pinwheel, Roll, Rotate. But none of those match the entry behaviour. 

Below I've modified the FadeOutDown animation to add `easeInOut` easing to make it feel closer to the timing animation for height.

https://github.com/Expensify/App/assets/6880914/d8e4f403-9b04-4de0-8b4f-a958b332d31a

Reply:
@shawnborton Thoughts on the above ^

Reply:
I think that feels fine to me, so let's go with that. Thanks for sharing!